### ISSUE TITLE
Circular dependency issues

### ISSUE THREAD

**Issue by o-alexandrov** [EOSIO/eosjs#969]
**Version of EOSJS**
22.0.0

**Describe the bug**

The issue was probably partially resolved in https://github.com/EOSIO/eosjs/pull/946
These are the remaining circular dependency issues when bundling `eosjs v22.0.0` with latest `rollup v2.52.3`:

```
Circular dependency: ../node_modules/eosjs/dist/eosjs-key-conversions.js -> ../node_modules/eosjs/dist/PublicKey.js -> ../node_modules/eosjs/dist/eosjs-key-conversions.js
Circular dependency: ../node_modules/eosjs/dist/eosjs-key-conversions.js -> ../node_modules/eosjs/dist/PublicKey.js -> ../node_modules/eosjs/dist/eosjs-key-conversions.js?commonjs-proxy -> ../node_modules/eosjs/dist/eosjs-key-conversions.js
Circular dependency: ../node_modules/eosjs/dist/eosjs-key-conversions.js -> ../node_modules/eosjs/dist/PrivateKey.js -> ../node_modules/eosjs/dist/eosjs-key-conversions.js
Circular dependency: ../node_modules/eosjs/dist/eosjs-key-conversions.js -> ../node_modules/eosjs/dist/Signature.js -> ../node_modules/eosjs/dist/eosjs-key-conversions.js
```

**Comment by bradlhart** [comment]
Ideally this should be resolved with #937.  Do you have a list of steps that I can check to confirm that this is the case?

**Comment by o-alexandrov** [comment]
@bradlhart I apologize for missing your message here.

I checked latest `develop` with changes from #937 merged into `develop` on `Sep 7th, 2021`.
- eosjs version at the time of new testing was `22.2.0`

Thank you, the circular dependency issue is resolved.
